integration_tests/e2e/searchForAPrisoner.cy.ts

```diff
@@ -188,6 +188,7 @@ context('Search for a prisoner', () => {
         .then(pageTitle => {
           cy.task('stubGetBookings', prisonerNumber)
           cy.task('stubGetOffender', offender)
+          cy.task('stubGetPrisonerById', { prisonerNumber, currentIncentive: { level: { description: 'Standard' } } })
```

Comment:
Intending to revise how these stubs and the mock data works for the integration tests as part of VB-729.